misc/msvs/props.template

```diff
@@ -5,6 +5,7 @@
     <NMakeReBuildCommandLine>%%REBUILD%%</NMakeReBuildCommandLine>
     <NMakeCleanCommandLine>%%CLEAN%%</NMakeCleanCommandLine>
     <NMakeOutput Condition="'$(NMakeOutput)' == ''">%%OUTPUT%%</NMakeOutput>
+    <LocalDebuggerCommand>$(NMakeOutput)</LocalDebuggerCommand>
```

Comment:
So this property is a property that VS usually writes to a `godot.vcxproj.user` file. This is because it's usually a property that the user often manually sets. I think it might be fine to be set here as a default, because any existing `godot.vcxproj.user` setting will override it, though. Have you tested overriding this in VS and seeing where it stores the override?